spec.html

```diff
@@ -32069,43 +32070,141 @@ <h1>Week Day</h1>
         <p>A weekday value of *+0*<sub>𝔽</sub> specifies Sunday; *1*<sub>𝔽</sub> specifies Monday; *2*<sub>𝔽</sub> specifies Tuesday; *3*<sub>𝔽</sub> specifies Wednesday; *4*<sub>𝔽</sub> specifies Thursday; *5*<sub>𝔽</sub> specifies Friday; and *6*<sub>𝔽</sub> specifies Saturday. Note that <emu-eqn>WeekDay(*+0*<sub>𝔽</sub>) = *4*<sub>𝔽</sub></emu-eqn>, corresponding to Thursday, 1 January 1970.</p>
       </emu-clause>
 
-      <emu-clause id="sec-local-time-zone-adjustment" type="implementation-defined abstract operation">
+      <emu-clause id="sec-getutcepochnanoseconds" type="abstract operation">
         <h1>
-          LocalTZA (
-            _t_: a Number,
-            _isUTC_: a Boolean,
-          ): an integral Number
+          GetUTCEpochNanoseconds (
+            _year_: an integer,
+            _month_: an integer between 1 and 12 inclusive,
```

Comment:
Is “inclusive” defined somewhere? as written I’m not sure if it includes 1 or 12 or both (i weakly lean towards both)

Reply:
I don't think it's defined, but there are many existing uses of it in ECMA-262 where it's clear that it's inclusive on both ends, e.g. [ToInt8](https://tc39.es/ecma262/#sec-toint8).

Reply:
Sounds good - @michaelficarra, perhaps we should define it in a separate PR?

Reply:
I think it's pretty intuitive and sufficiently unambiguous to our readers. I wouldn't be *against* adding a clarification of the term, but it'd probably take some convincing.

Reply:
I find it ambiguous :-) however your suggested change would make it unambiguous to me that "inclusive" applies to both ends, so that sounds great.

Reply:
I opened #2848. If we follow the convention I used in that PR, the phrasing here would be "an integer within the inclusive range from 1 to 12". We can wait for that to be merged before updating here, though.